server/server.js

```diff
@@ -20,9 +20,29 @@ server.listen(config.port);
 log.info('Server is on bruh and running on port: '+ config.port);
 
 //----------------------------------
+// Allow Cors Domain:
+//   res.header('Access-Control-Allow-Origin', 'MyDomainHere');
+//   This Allow for all client acces to your server
+//   Change for your IP or Domain or use localhost !
+//----------------------------------
+var allowCrossDomain = function(req, res, next) {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
+    res.header('Access-Control-Allow-Headers', 'Content-Type, x-access-token');
+    if (req.method === "OPTIONS") 
+        res.send(200);
+    else 
+        next();
+}
+
+//----------------------------------
 // SET ROUTES FOR EXPRESS API HERE:
 //----------------------------------
 
+//Allow Cors
+
+app.use(allowCrossDomain);
```

Comment:
Could we wrap this in an if statement checking a config option that the developer could set? With that additional logic it is quite simple to turn this on or off.